config.py.sample

```diff
@@ -37,6 +37,9 @@ MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
 LASTFM_API_KEY = "USE_LASTFM_API_KEY"
 
 
+# Max time in seconds after which the playing_now stream will expire.
+PLAYING_NOW_MAX_DURATION = 5 * 60
```

Comment:
5 minutes is not enough:

  https://musicbrainz.org/release/3591d22e-fdc5-309a-91d3-7bbafe086d6c/disc/1#7aefdb30-ad00-3cde-9072-4e43f4253637

A standard CD is 80 minutes. Let's set this to 80 minutes, which out to be sufficient for now.